docs/usage/distrobox-export.md

```diff
@@ -104,6 +104,29 @@ The option "--delete" will un-export an app, binary, or service.
 
 The option "--sudo" will launch the exported item as root inside the distrobox.
 
+**Exporting items from rootful containers**
```

Comment:
This should be removed as the work on export has been reverted

Reply:
Whoops, sorry.

Reply:
Update: I added a new section to replace this one with more useful instructions regarding the new implementation. Tell me if I should change anything :+1: